ingester/Dockerfile

```diff
@@ -11,7 +11,7 @@ WORKDIR /app/
 ADD pyproject.toml poetry.lock ./
 ADD download_embedder.py /app/
 
-RUN --mount=type=cache,target=$POETRY_CACHE_DIR poetry install --no-root --no-ansi --with worker,ingester --without ai,dev,api
+RUN --mount=type=cache,target=$POETRY_CACHE_DIR poetry install --no-root --no-ansi --with ingester,api --without ai,dev
```

Comment:
`api` seems like a big dependency to have for the ingester, are there some packages we could put in the ingester block instead?

Reply:
unfortunately ECS requires us to have an http endpoint, so we need all 3 dependancies in the api group in `core-api`, `embedder` and `ingester`